client-v2/src/components/util/ArticlePageNumberSection.tsx

```diff
@@ -0,0 +1,21 @@
+import CardMetaContent from '../../shared/components/card/CardMetaContent';
+import { CapiArticle } from 'types/Capi';
+import * as React from 'react';
+
+interface ComponentProps {
+  article?: CapiArticle;
```

Comment:
Could we move this nullability up? That is, I think this component could be simplified if it has an article to render against.

Reply:
can you tel more details what you mean :) ?

I do nullability here as the only logic that this component does is: if article is not defined do not display  `CardMetaContent` with page number, display otherwise

It is very easy to unit test 

If i move it up i will have to test it by testing `FeedItem.tsx` which is much larger and does not have unit tests as far as i can see for now

Reply:
resolved in person :)